File: snippets/code/web3py-contract/get.py
from compile import abi, bytecode
from web3 import Web3

provider_rpc = {
    'development': 'http://localhost:9933',
    'alphanet': 'https://rpc.api.moonbase.moonbeam.network',
}
web3 = Web3(Web3.HTTPProvider(provider_rpc["development"]))  # Change to correct network

contract_address = 'CONTRACT-ADDRESS-HERE'

print(f'Making a call to contract at address: { contract_address }')

Incrementer = web3.eth.contract(address=contract_address, abi=abi)
number = Incrementer.functions.number().call()

print(f'The current number stored is: { number } ')

File: .snippets/code/ethers-contract-local/get.js
const ethers = require('ethers');
const { abi } = require('./compile');

const providerRPC = {
  development: {
    name: 'moonbeam-development',
    rpc: 'http://localhost:9933',
    chainId: 1281,
  },
  moonbase: {
    name: 'moonbase-alpha',
    rpc: 'https://rpc.api.moonbase.moonbeam.network',
    chainId: 1287,
  },
};
const provider = new ethers.providers.StaticJsonRpcProvider(providerRPC.development.rpc, {
  chainId: providerRPC.development.chainId,
  name: providerRPC.development.name,
}); //Change to correct network

const contractAddress = 'CONTRACT-ADDRESS-HERE';

const incrementer = new ethers.Contract(contractAddress, abi, provider);

const get = async () => {
  console.log(`Making a call to contract at address: ${contractAddress}`);

  const data = await incrementer.number();

  console.log(`The current number stored is: ${data}`);
};

get();